bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/device.xml

```diff
@@ -102,7 +102,7 @@
 		<tags>
 			<tag>CurrentTemperature</tag>
 		</tags>
-		<state readOnly="true" pattern="%f %unit%">
+		<state readOnly="true" pattern="%d %unit%">
```

Comment:
This will produce warnings if fed by a float. Better use %.0f

Reply:
changed for all %d